build/Dockerfile.debug

```diff
@@ -2,4 +2,4 @@ FROM turbonomic/kubeturbo:6.4.0
 COPY kubeturbo.debug /opt/turbonomic/bin/kubeturbo
```

Comment:
Shall we change `turbonomic/kubeturbo:6.4.0` to `turbonomic/kubeturbo` ?

Reply:
yes, that makes sense, I will include it in this PR

Reply:
actually, Michael made the debug tag configurable, so I think we are good

Reply:
The tag `${TURBO_REPO}/kubeturbo:${KUBE_VER}debug .` defines the output image which is used for pushing the debug image to personal registry. The base image is still hard coded as `turbonomic/kubeturbo:6.4.0` which seems odd. But it doesn't matter in the end, as the 6.4.0 binary is replaced by `kubeturbo.debug`.

Reply:
that makes sense, changed it